src/antsibull/cli/antsibull_docs.py

```diff
@@ -347,7 +348,7 @@ def parse_args(program_name: str, args: List[str]) -> argparse.Namespace:
     if '--skip-indexes' in args:
         flog.warning('The CLI parameter, `--skip-indexes` has been renamed to'
                      ' `--no-indexes`.  Please use that instead')
-        if '--indexes' or '--no-indexes' in args:
+        if '--indexes' in args or '--no-indexes' in args:
```

Comment:
the other things in here seem like changes that just make the linting happy for the most part, but this looks like an actual bug was fixed? Or does `'string1' or 'string2' in args` actually check for both being in `args`? I would be very surprised if that were the case (but it would make sense why/how that would have gotten caught in linting, being flagged for being unclear/unintuitive).

Anyway, if it was an actual bug, it probably deserves a note in the PR description at least, probably doesn't warrant a separate issue filed though.

Reply:
I decided to move this into a separate PR: #377

(I originally wanted to do that anyway, but then forgot about it in the hours between working on this and creating the PR :) )